src/modules/main.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable  */
```

Comment:
I noticed you disabled eslint in your main.js file it is against mircroverse policy to disable linters because linters help us write clean code. it would be nice if you enabled eslint.